slack-triage-agent/main_polling.py

```
@@ -0,0 +1,732 @@
+                                alt_messages = result.data.get('messages', [])
+                            elif isinstance(result, dict):
+                                alt_messages = result.get('messages', [])
+                        print(f"   ↻ Fallback fetched {len(alt_messages)} messages")
+                        # Use whichever list is non-empty (prefer fallback if it found any)
+                        if alt_messages:
+                            messages = alt_messages
+                            # Set boundary based on newest message like fetch_channel_messages
+                            try:
+                                max_ts = max(float(m.get('ts')) for m in messages if m.get('ts'))
+                                last_poll_time[channel_id] = max_ts - float(Settings.POLL_OVERLAP_SECONDS)
+                            except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)